python/aswfdocker/tests/test_builder.py

```diff
@@ -0,0 +1,121 @@
+import unittest
+import logging
+import tempfile
+
+from click.testing import CliRunner
+
+from aswfdocker import builder, buildinfo, constants
+from aswfdocker.cli import aswfdocker
+
+
+class TestBuilder(unittest.TestCase):
```

Comment:
It looks like test_builder.py is hard coded to use 2019 as the test case, should that be configurable so it can easily be moved forward / backwards, or even across all supported years if you wanted to be completely thorough?

Reply:
I've been thinking about this a bit and the added complexity scares me more than having to update some tests in 2 years time...